samples/BUILD.gn

```diff
@@ -55,6 +55,7 @@ executable("pdfium_test") {
     "../:test_support",
     "//build/win:default_exe_manifest",
   ]
+  libs = [ "/usr/lib64/libjpeg.a"]
```

Comment:
Actually this doesn't make a lot of sense because pdfium_test doesn't even support JPEG output, but I got linker errors without this. Oh well.